Assets/Assets.csproj

```diff
@@ -1,19 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
It looks like these indents have changed from spaces to tabs. I think we should use spaces everywhere.

Reply:
Fixed